control-plane/msp-operator/src/main.rs

```diff
@@ -888,7 +888,15 @@ async fn pool_controller(args: ArgMatches<'_>) -> anyhow::Result<()> {
     let msp: Api<MayastorPool> = Api::namespaced(k8s.clone(), namespace);
     let lp = ListParams::default();
     let url = Url::parse(args.value_of("endpoint").unwrap()).expect("endpoint is not a valid URL");
-    let cfg = clients::tower::Configuration::new(url, Duration::from_secs(5), None, None, true)
+
+    let timeout: Duration = args
+        .value_of("request-timeout")
+        .unwrap()
```

Comment:
can we unwrap_or() use this https://github.com/DataCoreSoftware/bolt-control-plane/blob/2f8b0fccbfac994747fec65a81c04dd45790e9f0/utils/utils-lib/src/constants.rs#L4 ?

Reply:
Actually, I missed the bit where it's configuring allowed args: https://github.com/openebs/mayastor-control-plane/blob/develop/control-plane/msp-operator/src/main.rs#L954-L1008

I think it would be better to put this back to an `.unwrap()` here, and add:
```
        .arg(
            Arg::with_name("request-timeout")
                .short("t")
                .long("request-timeout")
                .env("REQUEST_TIMEOUT")
                .default_value(utils::DEFAULT_REQ_TIMEOUT)
                .help("the timeout for remote requests"),
        )
```
WDYT?